recipes/messenger/webview.js

```diff
@@ -126,4 +126,11 @@ module.exports = (Ferdium, settings) => {
     },
     true,
   );
+
+  if (
+    window.matchMedia &&
+    window.matchMedia('(prefers-color-scheme: dark)').matches
```

Comment:
We should not be using this but instead use the internal config of the service for handling dark mode (I'm on mobile right now but should be something like service.isDarkMode)